about/policy/metadata/index.md

```diff
@@ -0,0 +1,113 @@
+---
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+tags:
+  - metadata
+  - sgid
+date: 2020-07-09 09:33:34 -0600
+title: Basic Metadata Guidelines
+categories: []
+---
+
+By most accounts, creating metadata is right up there with organizing your garage. It can be a chore, but it doesn't have to be.
+
+At AGRC, we have decided to make the process more conversational. In other words, instead of looking at a long list of metadata requirements, we thought it would be easier to talk about the data. This is our solution to metadata procrastination.
+
+We have distilled the basic elements of metadata into a few approachable questions. If you can answer these simple questions about your data, then you're officially on your way to good metadata.
+
+#### Can you answer these questions about your data?
+{: .text-left}
+
+##### Summarize your Data
+{: .text-left}
+
+- What is this dataset?
+- What is the purpose of this dataset?
+
+##### Describe your Data
+{: .text-left}
+
+- What does this dataset represent?
+  - ex: road centerlines represent the centerline of a physical road
+- How was this dataset created?
+  - ex: digitized, GPS, etc.
+- How reliable and accurate is the dataset?
+  - ex: survey-grade, mapping-grade, etc.
+- What is the general update schedule for this dataset?
+  - ex: monthly, yearly, as needed, etc.
+- Are there any warnings or limitations on the data?
+  - ex: don’t use parcels as a legal boundary depiction
+  - ex: don’t use parcels as a complete dataset for geocoding
+  - ex: don’t use zip codes for demographic generalization
+- Are there any common misconceptions or assumptions about this data that should be further explained?
+
+##### Establish Contact Info
+{: .text-left}
+
+- Who created this dataset?
+- What is the best point of contact for this dataset?
+  - if someone had further questions or would like to get involved, who should they contact?
+
+Your answers to these questions can be plugged into the basic metadata elements (discussed below). It's that simple - answer a few questions, transfer your responses, and voilà, you officially have informative, standardized metadata.
+
+*Note: If you contribute data to Utah's State Geographic Information Database (SGID), we encourage you to use our [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit). It's a great way to update your metadata and make it more discoverable. The form covers all the questions above -and best of all- after you submit your responses, your portion of the process is done! AGRC, in collaboration with the DTS editor, will handle the remaining details.*
```

Comment:
**chore**: the convention for italics on our website is a `_`. Consult the readme for other conventions.  
```suggestion
_Note: If you contribute data to Utah's State Geographic Information Database (SGID), we encourage you to use our [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit). It's a great way to update your metadata and make it more discoverable. The form covers all the questions above -and best of all- after you submit your responses, your portion of the process is done! AGRC, in collaboration with the DTS editor, will handle the remaining details._
```

Reply:
oops, busted!  thx